r/datascience/requirements.R

```diff
@@ -0,0 +1,23 @@
+arrow
+caret
+crayon
+devtools
+dplyr
+e1071
+forecast
+ggplot2
+ggpubr
+hexbin
+htmltools
+htmlwidgets
+nycflights13
+randomforest
+rcurl
```

Comment:
Maybe a silly question but do we need to include `reticulate` and the IRkernel dependency stuff in here too?